src/Reddit/Reddit.ts

```diff
@@ -105,50 +421,32 @@ export default <IPluginBase & IPlugin>{
     },
 
     init: async () => {
-      // there is a global command, so init runs everywhere
-      if (document.location.hostname.endsWith("reddit.com")) {
+      if (location.hostname.endsWith("reddit.com")) {
         console.log("init");
 
-        if (/^https?:\/\/www.reddit/.test(document.location.href)) {
-          document.location.href = document.location.href.replace(
-            /^https?:\/\/.*\.reddit.com/,
-            "http://old.reddit.com"
-          );
-        }
-
-        if (COMMENTS_REGX.test(document.location.href)) {
-          PluginBase.util.prependContext("Post");
-          PluginBase.util.removeContext("Post List");
-        } else {
-          PluginBase.util.prependContext("Post List");
-          PluginBase.util.removeContext("Post");
-        }
+        isOldReddit = /https:\/\/old/.test(location.href);
```

Comment:
how will this work if the user has the setting to use the old style reddit, and is on www.reddit.com?

Reply:
we can remove it, anyway it works fine